tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -1337,4 +1337,13 @@ class WskBasicUsageTests
             }
             tmpProps.delete()
     }
+
+    behavior of "Wsk activation"
+
+    it should "reject activation list with a limit that exceeds the maximum limit" in {
+        val maxLimit = 200
```

Comment:
Is a configurable openwhisk limit - that is, it's available in whisk.properties or other configuration file?